Source/Scene/ModelExperimental/ModelExperimentalUtility.js

```diff
@@ -212,13 +212,25 @@ ModelExperimentalUtility.correctModelMatrix = function (
   forwardAxis
 ) {
   if (upAxis === Axis.Y) {
-    Matrix4.multiplyTransformation(modelMatrix, Axis.Y_UP_TO_Z_UP, modelMatrix);
+    modelMatrix = Matrix4.multiplyTransformation(
```

Comment:
Hm, if you're going to update this function, let's make it more consistent with the typical style by making `modelMatrix` a true result parameter (that should get returned). By the end it should look like this:

```
const correctedMatrix = ModelExperimentalUtility.correctModelMatrix(upAxis, forwardAxis, result);
```